web/js/components/location-search/coordinates-dialog.js

```diff
@@ -22,8 +26,13 @@ class CoordinatesDialog extends Component {
     setTimeout(() => {
       this.setState({ showTooltips: true });
     }, 200);
+    events.on('location-search:coordinate-format', this.updateCoordinateFormat);
```

Comment:
We should also deregister this event when the component unmounts.  The call to deregister is basically the same as this just with `event.off()` instead of `event.on()`.

Reply:
Added a componentWillUnmount method to deregister the event.